go/exorelay/exorelay_test.go

```diff
@@ -27,16 +29,51 @@ func newExocom() *exocomMock.ExoComMock {
 	return exocom
 }
 
+func WaitFor(condition func() bool, errorMessage string) error {
```

Comment:
I don't like the look of that library. It has retry limit and has you introduce your own delays. Our implementation has a timeout with a delay built in.